src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -638,6 +678,56 @@ void MenuGame::initInput()
             }
         },
         t::Once);
+    game.addInput(
+        {{k::F5}},
+        [this](ssvu::FT /*unused*/) { reloadLevelAssets(); },t::Once);
+}
+
+void MenuGame::reloadLevelAssets()
+{
+    if(state != States::SMain || !dialogText.empty() || !Config::getDebug()) return;
+    
+    noActions = 2;
```

Comment:
This line needs to be more self-explanatory or needs a comment. Why `2`? What does this mean in practice?